test/testdata/lsp/rename/method_attr.rb

```diff
@@ -0,0 +1,10 @@
+# typed: true
+# frozen_string_literal: true
+
+class Foo
+  attr_reader :bar
+end
+
+f = Foo.new
+f.bar
+# ^ apply-rename: [A] newName: baz
```

Comment:
Note that this won't completely work since `attr_reader :bar` also creates a `@bar` in the class, which the class assigns to to populate the reader. But that's an acceptable limitation for now; we'd need the ability 1) to recognize it's an `attr_reader` and 2) to rename the field.

Reply:
Yeah... good point. Do you think we should explicitly check for this and reject the rename? Or is it enough just to document this as a current limitation?

Reply:
I'm OK with it as a current limitation.

Reply:
Updating the field references is much easier than the method references, as the field can only be referenced within the class methods.